backstop/tests/text-field.html

```diff
@@ -29,6 +29,16 @@
         flex-direction: column;
         gap: 11px;
       }
+
+      #block-labels > .iui-input-container { margin-bottom: 11px; }
```

Comment:
Move CSS rules to a new line. Same for bellow one.

Reply:
Done